agg-ast/ast/src/definitions.rs

```diff
@@ -1278,6 +1261,28 @@ pub enum TaggedOperator {
     Shift(Shift),
     #[serde(rename = "$cond")]
     Cond(Cond),
```

Comment:
These made sense to just add as TaggedOperators, fun thing is they all use the same underlying struct